### PR TITLE
Migrate podman inspect and tag to image library

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: libpod/image/image.go [hunk 1]
```diff
@@ -226,11 +234,20 @@ func (i *Image) ID() string {
 	return i.image.ID
 }
 
-// Digest returns the image's Manifest
+// Digest returns the image's digest
 func (i *Image) Digest() digest.Digest {
 	return i.image.Digest
 }
 
+// Manifest returns the images manifest and manifest type
```
Comment (mheon) [inline libpod/image/image.go]: Can this get a better comment? Is the manifest the []byte or the string?

**Comment by mheon** [comment]
One nit otherwise LGTM

**Comment by baude** [comment]
bot, retest this please

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**